pallets/automation-time/src/lib.rs

```diff
@@ -429,9 +425,11 @@ pub mod pallet {
 					},
 				};
 
-				weight_left = weight_left - cost - RUN_TASK_LOOP_OVERHEAD;
+				// need to calculate the look cost minus the action
+				weight_left = weight_left - action_weight - 10_000;
 
-				if weight_left < MAX_LOOP_WEIGHT {
+				// need to calculate the max cost of the loop
+				if weight_left < 20_000 {
```

Comment:
at some point once it's locked it, you'll have to pull out these weights and static numbers so that we can all understand them

Reply:
all of these values will eventually be pulled from the weights file